specification/resources/resource-manager/Microsoft.Authorization/stable/2019-06-01/examples/createPolicyAssignment.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "scope": "subscriptions/ae640e6b-ba3e-4256-9d62-2993eecfa6f2",
+    "policyAssignmentName": "EnforceNaming",
+    "api-version": "2019-06-01",
+    "parameters": {
+      "properties": {
+        "displayName": "Enforce resource naming rules",
+        "description": "Force resource names to begin with given DeptA and end with -LC",
+        "metadata": {
+          "assignedBy": "Special Someone"
+        },
+        "policyDefinitionId": "/subscriptions/ae640e6b-ba3e-4256-9d62-2993eecfa6f2/providers/Microsoft.Authorization/policyDefinitions/ResourceNaming",
+        "parameters": {
+          "prefix": {
+            "value": "DeptA"
+          },
+          "suffix": {
+            "value": "-LC"
+          }
+        },
+        "enforcementMode": "DoNotEnforce"
```

Comment:
>"enforcementMode": "DoNotEnforce" [](start = 8, length = 33)

This isn't necessarily blocking, but you should probably add a new example with DoNotEnforce instead of setting that in the "default" example. Maybe call the new example createPolicyAssignmentWithoutEnforcement or something like that?